WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -380,10 +380,14 @@ extension AuthenticationManager: WordPressAuthenticatorDelegate {
     ///
     func presentSupport(from sourceViewController: UIViewController, sourceTag: WordPressSupportSourceTag) {
         let identifier = HelpAndSupportViewController.classNameWithoutNamespaces
-        guard let supportViewController = UIStoryboard.dashboard.instantiateViewController(withIdentifier: identifier) as? HelpAndSupportViewController else {
-            return
-        }
+        let supportViewController = UIStoryboard.dashboard.instantiateViewController(identifier: identifier,
+                                                                                     creator: { coder -> HelpAndSupportViewController? in
+            guard let customURL = sourceTag.customHelpCenterURL else {
+                return nil
```

Comment:
I'm a bit confused here, does `nil` mean that it's not a custom instance? it feels like it might be returning a`nil` view controller for the support UI. maybe a comment would help?

Reply:
also, how about we pass the `sourceTag` to `HelpAndSupportViewController` instead of a calculated custom URL? then the logic about the custom URL can be inside `HelpAndSupportViewController`, which is its implementation detail.

Reply:
> I'm a bit confused here, does nil mean that it's not a custom instance? it feels like it might be returning anil view controller for the support UI. maybe a comment would help?

I added a comment in https://github.com/woocommerce/woocommerce-ios/pull/7553/commits/12e032952b6951b0cce4ae7decb9bec213dabd90. Kindly take a look and let me know if you think I can add more details. 🙏

Reply:
> also, how about we pass the sourceTag to HelpAndSupportViewController instead of a calculated custom URL? then the logic about the custom URL can be inside HelpAndSupportViewController, which is its implementation detail.

I considered this, but we will be presenting custom help content for screens that aren't from the WordPressAuthenticator library. This means that those screens will not have a support source tag. So, I felt that it is better to keep the source tag knowledge out of `HelpAndSupportViewController`. 

I also considered hiding the URL logic inside a private extension to `WordPressSupportSourceTag` inside `AuthenticationManager` file. But, this means that I cannot unit test the `customHelpCenterURL` property. 🤔 

Kindly let me know your thoughts.